packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -24,9 +26,52 @@ const errors = {
 // TODO: Read this from the chain via rpc.
 const CREATION_FEE = '10 BLD';
 
-const ProvisionDialog = ({ onClose, open, address, href, keplrConnection }) => {
+// 100 IST
+const MINIMUM_PROVISION_POOL_BALANCE = 100_000_000n;
+
+/** @typedef {import('@agoric/vats/src/provisionPool.js').MetricsNotification} ProvisionPoolMetrics */
```

Comment:
the lint:types thing will require shaving many yaks. for now:
```suggestion
// XXX import from the contract
/**
 * @typedef {object} ProvisionPoolMetrics
 * @property {bigint} walletsProvisioned  count of new wallets provisioned
 * @property {Amount<'nat'>} totalMintedProvided  running sum of Minted provided to new wallets
 * @property {Amount<'nat'>} totalMintedConverted  running sum of Minted
 * ever received by the contract from PSM
 */
```

Reply:
Done, thanks